modules/4337/certora/conf/SignatureLengthCheck.conf

```diff
@@ -0,0 +1,15 @@
+{
+    "files": [
+        "certora/harnesses/Safe4337ModuleHarness.sol",
+        "certora/harnesses/Account.sol",
+    ],
+	"optimistic_loop": true,
```

Comment:
Have you tried with different values of `loop_iter`? Default is 1.
Would be nice to use value 2 or 3 if possible.
https://docs.certora.com/en/latest/docs/prover/cli/options.html#loop-iter

Reply:
Updated, and helped to figure out a counter-example with the first rule (though curious why it worked with just one iteration, still investigating), so had to remove it, as we have a stronger rule with the canonical hash one.